superset/connectors/sqla/models.py

```diff
@@ -226,8 +226,8 @@ class TableColumn(Model, BaseColumn, CertificationMixin):
     table_id = Column(Integer, ForeignKey("tables.id"))
     table: SqlaTable = relationship(
         "SqlaTable",
-        backref=backref("columns", cascade="all, delete-orphan"),
-        foreign_keys=[table_id],
```

Comment:
Per the [docs](https://docs.sqlalchemy.org/en/14/orm/relationship_api.html#sqlalchemy.orm.relationship.params.foreign_keys) it states that this is not normally needed.